tests/csrf_tests/tests.py

```diff
@@ -17,6 +18,53 @@
     post_form_view, token_view,
 )
 
+# This is a test (unmasked) CSRF cookie / secret.
+TEST_SECRET = 'lcccccccX2kcccccccY2jcccccccssIC'
+# Generate two masked versions of TEST_SECRET for testing purposes.
+MASKED_TEST_SECRET1 = '1bcdefghij2bcdefghij3bcdefghij4bcdefghij5bcdefghij6bcdefghijABCD'
+MASKED_TEST_SECRET2 = '2JgchWvM1tpxT2lfz9aydoXW9yT1DN3NdLiejYxOOlzzV4nhBbYqmqZYbAV3V5Bf'
+
+
+class CsrfFunctionTests(SimpleTestCase):
+
+    def test_unmask_cipher_token(self):
+        cases = [
+            (TEST_SECRET, MASKED_TEST_SECRET1),
+            (TEST_SECRET, MASKED_TEST_SECRET2),
+            (32 * 'a', 'vFioG3XOLyGyGsPRFyB9iYUs341ufzIEvFioG3XOLyGyGsPRFyB9iYUs341ufzIE'),
+            (32 * 'a', 64 * 'a'),
+            (32 * 'a', 64 * 'b'),
+            (32 * 'b', 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb'),
```

Comment:
I wonder if this and below wouldn't be more readable as `32 * 'a' + 32 * 'b'` but no strong feelings

Reply:
Thanks, @apollo13. That would be okay with me.

Reply:
Changed.